src/flask_restalchemy/api.py

```diff
@@ -257,6 +257,31 @@ def register_view(self, view_func, url, pk="id", pk_type="int", methods=None):
                     f"{url}/<{pk_type}:{pk}>", view_func=view_func, methods=methods
                 )
 
+    def route(self, rule, endpoint=None, **kwargs):
+        """
+        A decorator that is used to register a view function for a
+        given URL rule.  This does the same thing as :meth:`add_url_rule`
+        but is intended for decorator usage::
+
+            @api.route('/')
+            def index():
+                return 'Hello World'
+
+        :param rule: the URL rule as string
+        :param endpoint: the endpoint for the registered URL rule.  Flask
```

Comment:
The "Flask itself assumes..." sentence is a bit misleading, because we handle it, not flask. I suggest changing it to `Use the name of the view function by default`.

Reply:
done